client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -55,6 +59,12 @@ class CampaignAdminList extends Component {
         .then(() => {
           this.setBreadcrumbs();
           this.setState({ loading: false });
+        })
+        // Catch error and set Error code
+        .catch((e) => {
+          this.setState({ loading: false });
+          this.setState({ error: true });
+          this.setState({ errorCode: e.response.status });
```

Comment:
Can these be combined as a single call? e.g.
```js
this.setState({
  loading: false,
  error: true,
  errorCode: e.response.status,
});
```

Reply:
DONE